.pre-commit-config.yaml

```diff
@@ -56,4 +56,7 @@ repos:
               [.]pre-commit-config[.]yaml$|
               [.]pre-commit-hooks[.]yaml$|
               pyproject[.]toml$
+        exclude: |
+          (?x)
+              test_copyright[.]py$
```

Comment:
Why do you have to exclude this?

Reply:
Because the test cases themselves include copyright headers that shouldn't be updated. I could make them unrecognizable to the script by inserting string breaks, or I could just exempt the file from the check :)

Reply:
I can split this into a separate PR if you want.

Reply:
Oh interesting. That's fine, then.